### ISSUE TITLE
3.11a7 CTracer compilation fails

### ISSUE THREAD

**Issue by reaperhulk** [coveragepy/coveragepy#1367]
Under Python 3.11.0a7 the current coverage release fails to successfully compile the CTracer due to changes in the underlying PyCodeObject representation. https://github.com/python/cpython/pull/91666 is probably a good place to start looking into this if you weren't already aware of the changes.

```
coverage/ctracer/tracer.c: In function ‘CTracer_handle_call’:
coverage/ctracer/tracer.c:533:46: error: ‘PyCodeObject’ {aka ‘struct PyCodeObject’} has no member named ‘co_code’
  533 |     PyObject * pCode = MyFrame_GetCode(frame)->co_code;
      |                                              ^~
coverage/ctracer/tracer.c: In function ‘CTracer_handle_return’:
coverage/ctracer/tracer.c:701:54: error: ‘PyCodeObject’ {aka ‘struct PyCodeObject’} has no member named ‘co_code’
  701 |             PyObject * pCode = MyFrame_GetCode(frame)->co_code;
      |                                                      ^~
```

As always, thank you so much for the most valuable package in Python. 😄

**Comment by nedbat** [comment]
I've fixed the compilation errors in the [nedbat/three-eleven-beta](https://github.com/nedbat/coveragepy/tree/nedbat/three-eleven-beta) branch, but the results are wrong for generators.  I'm not sure what detail I'm missing.  @markshannon? @vstinner?

**Comment by vstinner** [comment]
See also my WIP PR: https://github.com/nedbat/coveragepy/pull/1353

**Comment by nedbat** [comment]
This is fixed in 956f0fde.

**Comment by nedbat** [comment]
This is now released as part of [coverage 6.3.3](https://pypi.org/project/coverage/6.3.3).
